src/main/java/org/broadinstitute/consent/http/util/DatasetUtil.java

```diff
@@ -1,31 +1,14 @@
 package org.broadinstitute.consent.http.util;
 
+import org.apache.commons.lang3.StringUtils;
+
 public class DatasetUtil {
 
     private static final String PREFIX = "DUOS-";
 
     public static String parseAlias(Integer aliasUid) {
-        String alias;
-        if(aliasUid < 10) {
-            alias = PREFIX + "00000" + aliasUid;
-        }
-        else if(aliasUid < 100) {
-            alias = PREFIX + "0000" + aliasUid;
-        }
-        else if(aliasUid < 1000) {
-            alias = PREFIX + "000" + aliasUid;
-        }
-        else if(aliasUid < 10000) {
-            alias = PREFIX + "00" + aliasUid;
-        }
-        else if(aliasUid < 100000) {
-            alias = PREFIX + "0" + aliasUid;
-        }
-        else {
-            alias = PREFIX + aliasUid;
-        }
+        String alias = PREFIX + StringUtils.leftPad(aliasUid.toString(), 6, "0");
         return alias;
-
     }
```

Comment:
Fantastic simplification. This also leaves me with the impression that this class is completely useless. The dataset class itself should just return this value from the current `getAliasParsed()` method.

Reply:
I think refactoring this class away might be beyond the scope of the PR, but if you're willing to take that one, please do. Otherwise, we should file a ticket for the refactoring.

Reply:
Sounds good I'll make a new ticket and PR, and remove the class there